app/services/user_registrator.rb

```diff
@@ -38,11 +38,11 @@ def register!(stripe_token, invite_token)
   end
 
   def stripe_payment!(stripe_token)
-    charge = StripeWrapper::Charge.create(
-      :amount => 999, # amount in cents, again
+    customer = StripeWrapper::Customer.create(
       :source => stripe_token,
-      :description => "payment of #{user.email}"
+      :user => user
```

Comment:
Why do you use old ruby hash syntax here? You have fixed it in your StripeWrapper.